ic-task-scheduler/src/scheduler.rs

```diff
@@ -684,7 +710,7 @@ mod test {
 
                     // beware that the the first execution is not a retry
                     for i in 1..=retries {
-                        scheduler.run().unwrap();
+                        scheduler.run(()).unwrap();
```

Comment:
Self::Ctx::default()

Reply:
Clippy warns about calling default() for unit type. Let's keep it as is.

Reply:
bad hint :(